### PR TITLE
Show mode and price for new courses

### PR DESCRIPTION
**Pull request by mikix**
I could have done a conditional check for the course version field, but I figured dumber was better here.  If they have an entitlement, they have a price & mode.

https://openedx.atlassian.net/browse/LEARNER-3769

### CODE REVIEW

**Review by jlajoie — APPROVED** [review]
:+1:

File: course_discovery/apps/publisher/templates/publisher/course_detail.html [hunk 1]
```diff
@@ -71,6 +71,30 @@
                 <div class="show-diff"></div>
             </div>
 
+            <div class="info-item history-field-container">
+                <div class="heading">
+                    {% trans "Enrollment Track" %}
```
Comment (amangano-edx) [inline course_discovery/apps/publisher/templates/publisher/course_detail.html:76]: Do these headings appear even if they don't have an entitlement?

File: course_discovery/apps/publisher/templates/publisher/course_detail.html [hunk 2]
```diff
@@ -71,6 +71,30 @@
...
+                    {% with object.entitlements.first.mode as field %}
+                        {% include "publisher/_render_optional_field.html" %}
+                    {% endwith %}
+                </div>
+                <div class="show-diff"></div>
+            </div>
+
+            <div class="info-item history-field-container">
+                <div class="heading">
+                    {% trans "Certificate Price" %}
```
Comment (amangano-edx) [inline course_discovery/apps/publisher/templates/publisher/course_detail.html:88]: Does this appear even without a price?

File: course_discovery/apps/publisher/templates/publisher/course_detail.html (hunk 1 above)
Comment (amangano-edx) [reply course_discovery/apps/publisher/templates/publisher/course_detail.html:76]: Also, translations should be escaped.

Comment (mikix) [reply course_discovery/apps/publisher/templates/publisher/course_detail.html:76]: Yes, the headings appear without an entitlement. Per the ACs, we are supposed to treat them like optional fields that aren't filled in yet.

Django templates should be autoescaped already.

File: course_discovery/apps/publisher/templates/publisher/course_detail.html (hunk 2 above)
Comment (mikix) [reply course_discovery/apps/publisher/templates/publisher/course_detail.html:88]: Yes, the headings appear without an entitlement. Per the ACs, we are supposed to treat them like optional fields that aren't filled in yet.

File: course_discovery/apps/publisher/templates/publisher/course_detail.html (hunk 1 above)
Comment (amangano-edx) [reply course_discovery/apps/publisher/templates/publisher/course_detail.html:76]: I just tested this, and they are not autoescaped.

Comment (mikix) [reply course_discovery/apps/publisher/templates/publisher/course_detail.html:76]: Right you are! As we discussed afterward in person, we'll file a ticket to address this module-wide.

**Pull request merged**